src/cluster-configuration/deploy/priority-class.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: pai-daemon-service-priority
```

Comment:
daemon and service are duplicate, pls keep only one term

Reply:
fixed